config/install/README.md

```diff
@@ -1,101 +1,591 @@
-# Install Kuadrant and Sail via OLM
+# Install and Configure Kuadrant and Sail via OLM using the kubectl CLI
+
+This document will walk you through setting up the required configuration to install kaudrant using [kustomize](https://kustomize.io/) or a tool that leverages kustomize such as kubectl along with OLM. It will also go through more advanced configuration options to enable building up a resilient configuration. You can view the full configuration built here: [Full AWS Example](https://github.com/Kuadrant/kuadrant-operator/tree/main/config/install/full-example-aws).
+
+
+
+1. [Basic Install](#basic-installation)
+
+2. [Configure DNS and TLS integration](#configure-dns-and-tls-integration)
+
+3. [External Redis for Rate Limit Counters](#use-an-external-redis)
+
+4. [Limitador Resilient Configuration](#limitador-topologyconstraints-poddisruptionbudget-and-resource-limits)
+
+5. [Authorino Resilient Configuration](#authorino-topologyconstraints-poddisruptionbudget-and-resource-limits)
+
+4. [[OpenShift Specific] Setup Observability ](#set-up-observability-openshift-only)
+
 
 ## Prerequisites  
-- Clone the[ Kuadrant-operator](https://github.com/Kuadrant/kuadrant-operator) repo
-- OLM (operator lifecycle manager)
-- cert-manager 
+- OCP or K8s cluster and CLI available.
+- OLM installed [operator lifecycle manager releases](https://github.com/operator-framework/operator-lifecycle-manager/releases)
+- (Optional) Gateway Provider Installed: By default this guide will install the [Sail Operator](https://github.com/istio-ecosystem/sail-operator) that will configure and install an Istio installation. Kuadrant is intended to work with [Istio](https://istio.io) or [Envoy Gateway](https://gateway.envoyproxy.io/) as a gateway provider before you can make use of Kuadrant one of these providers should be installed.  
```

Comment:
```suggestion
```
To avoid confusion, I'd remove this optional dependency because the guide shows how to install kuadrant with istio/sail. The 2 are entwined in the steps below, such that if the user already has a gateway provider installed that they want to use, there is extra work to decouple the istio/sail pieces.

Alternatively, I'm wondering if we should decouple those steps for the user anyways in this guide?
I think it's reasonable that a user could arrive at the Kuadrant installlation guide with or without a gateway provider already running in their cluster.

Reply:
Yeah but perhaps we can add that as an follow up. I think there is a lot of value in the guide as is for now?